lepton/manifest.go

```diff
@@ -418,6 +418,28 @@ func (m *Manifest) String() string {
 	//
 	if len(m.klibs) > 0 {
 		sb.WriteString("klibs:bootfs\n")
+
+		for _, klib := range m.klibs {
+			if klib == "ntp" {
+
+				ntpAddress := "pool.ntp.org"
+				ntpPort := "123"
+
+				if val, ok := m.environment["ntpAddress"]; ok {
+					ntpAddress = val
+				}
+
+				if val, ok := m.environment["ntpPort"]; ok {
+					ntpPort = val
+				}
+
+				sb.WriteString(fmt.Sprintf("ntp_address:%s\n", ntpAddress))
+				sb.WriteString(fmt.Sprintf("ntp_port:%s\n", ntpPort))
```

Comment:
pool.ntp.org and port number 123 are used by default in the kernel if nothing is specified in the manifest, so maybe I would not put the ntp_address and ntp_port parameters in the manifest unless the user explicitly specifies a value for these options.
As a side note, requiring the user to pollute the environment with parameters that are not used as environment variables isn't ideal, it seems to me it would be better to have such parameters separate from the environment; but that requires the implementation of something like ManifestVariables (https://github.com/nanovms/ops/issues/878).

Reply:
The issue with ManifestVariables is the user can pollute the manifest file with whichever variable he wants. Not sure of the implications of this. However, we can limit the variables used in ManifestVariables in ops.

@francescolavra @eyberg any thoughts on this?

Reply:
I see problems with both approaches but I think either way is fine for now - we can refactor this later